charts/dynamic-link-server/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: dynamiclinkserver
+description: A Helm chart for Kubernetes
```

Comment:
Please, provide a better description. You can take it from the initial repo

Reply:
Wrote a better description